### DISCUSSION TITLE
Wayland / Electron 12

### DISCUSSION THREAD

**Discussion by blauch** [atom/atom#22621] [General]
Hi There

I am using Atom for years and am very happy with it. I recently switched to sway and want to run apps rather fully native to wayland. Electron 12 supports this by adding flags `--enable-features=UseOzonePlatform --ozone-platform=wayland`. If I am not mistaken then Atom still relies on Electron 9. When will Atom be moving to Electron 12?

**Comment by ThatXliner** [comment]
From personal experience, using tree sitters is hard with electron version <10. Because native modules. (Unless I'm mistaken)
